tmdb_client.py

```diff
@@ -0,0 +1,61 @@
+import requests
+
+api_token = "eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJlM2QxYTRlZTEzODYyNDFmNWZiOTYxNzNiNzMyNDgzNyIsInN1YiI6IjYxOTNlZmY2OTU3ZTZkMDA4NWMzMzA3OSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.1L5qiruEWUs5xNQMLrDHW7yRTPGRBh1LBGGbCCbelSc"
```

Comment:
nie powinnaś umieszczać tokena w repozytorium. To jest zła praktyka jeśli chodzi o bezpieczeństwo. Spróbuj wczytać token z pliku albo (najlepsze rozwiązanie, ale musiałabyś trochę więcej poczytać ) użyć zmiennych środowiskowych (https://www.nylas.com/blog/making-use-of-environment-variables-in-python/)